dev/tasks/dev/utils/git.js

```diff
@@ -142,5 +144,26 @@ module.exports = {
 		];
 
 		tools.shExec( commitCommands.join( ' && ' ) );
+	},
+
+	/**
+	 * Adds remote to repository under specified path.
+	 *
+	 * @param {String} repositoryPath
+	 * @param {String} repositoryUrl
+	 */
+	addRemote( repositoryPath, repositoryUrl ) {
```

Comment:
That's a wrong argument name. You get `gitHubUrl` and you turn it into `repositoryUrl`.